src/scss/components/_header.scss

```diff
@@ -0,0 +1,95 @@
+.header {
+  position: sticky;
+  top: 0;
+  background: var(--white);
+
+  &__container {
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
+    margin: 0 auto;
+    max-width: 1440px;
```

Comment:
Not sure if the header should be tight to 1440px max-widht, generally it is not the case i.e https://www.blog.youtube, maybe we will need to confirm

Reply:
The  `max-width: 1440px` is to limit the objects inside this container, but the white background is full bleed